src/Core/Entity/HXL/HXLTagAttributes.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Ushahidi HXLTag Entity
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Entity\HXL;
+
```

Comment:
I don't think we need an entity and repo for this since we only really use the types and it's not an entity in HXL , but i'm not sure.